spec/cypress/app_commands/scenarios/lead_provider_with_schools.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+profile = LeadProviderProfile.last
```

Comment:
I _think_ there will only be one, but it would be safer to add an `order(:created_at)` here, since we use UUID primary keys

Reply:
👍